core/src/main/resources/org/apache/spark/ui/static/executorspage.js

```diff
@@ -126,7 +126,6 @@ function totalDurationAlpha(totalGCTime, totalDuration) {
     (Math.min(totalGCTime / totalDuration + 0.5, 1)) : 1;
 }
 
-// When GCTimePercent is edited change ToolTips.TASK_TIME to match
```

Comment:
hmm, why do you propose to remove this?

Reply:
ToolTips.TASK_TIME - Shaded red when garbage collection (GC) time is over 10% of task time

- Adding this variable for the first time
CommitID: c037d25482ea63430fb42bfd86124c268be5a4a4 
JIRA: https://issues.apache.org/jira/browse/SPARK-12149
[SPARK-12149][WEB UI] Executor UI improvement suggestions - Color UI
PR: https://github.com/apache/spark/pull/10154
First declaration：https://github.com/apache/spark/pull/10154/files#diff-91f8d288f7aa0726fc067ad9eb67aff41c474ada4d702e5b393bd81ae5aee7b5R91-R92
First use：https://github.com/apache/spark/pull/10154/files#diff-3bb7f9962a6343cbc09be474b85f2f9ed2e80047f4863d447faa407d925f425cR82


- The variable content is directly hard coding in `executorspage-template.html` file
CommitID: b9bab4dcf6cec5ec9e9860871f9dd1ec633a1d22
JIRA: https://issues.apache.org/jira/browse/SPARK-15951
[SPARK-15951] Change Executors Page to use datatables to support sorting columns and searching
PR: https://github.com/apache/spark/pull/13670
The variable content is directly hard coding: https://github.com/apache/spark/pull/13670/files#diff-cc6993f75c03b53bb41d8d7113ed3d5157580e583672da68b15cbe4d9f374e15R36
Delete: https://github.com/apache/spark/pull/13670/files#diff-3bb7f9962a6343cbc09be474b85f2f9ed2e80047f4863d447faa407d925f425cL94


**So I think it needs to be removed; Meanwhile, I did not find any usage in the `spark` code base using the following command: find . -name "*" -type file -print0 | xargs -0 grep "TASK_TIME"**